sdk/playwrighttesting/ci.yml

```diff
@@ -0,0 +1,41 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
```

Comment:
We'll also need a `tests.yml` file and a set of test resources to enable live testing.

Example:
  - [AppConfig](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/appconfiguration)

Reply:
We require live tests for any library that works against an Azure service.   Any dispensation is a conversation between yourself and the architects.